src/plugins/intel_gpu/src/graph/include/dft_inst.h

```diff
@@ -10,15 +10,6 @@
 
 namespace cldnn {
 
-template <>
-struct typed_program_node<dft> : public typed_program_node_base<dft> {
```

Comment:
can we do this refactoring in sepate PR, as it's not connected with MatrixNms operation

Reply:
done